inc/commonitilobject.class.php

```diff
@@ -8463,7 +8465,17 @@ public function handleTaskTemplateInput() {
       $itiltask   = new $itiltask_class;
       foreach ($this->input['_tasktemplates_id'] as $tasktemplates_id) {
          $tasktemplate->getFromDB($tasktemplates_id);
-         $tasktemplate_content = Toolbox::addslashes_deep($tasktemplate->fields["content"]);
+
+         // Parse twig template
+         $parameters_class = self::getUserTemplatesParametersClass();
+         $parameters = new $parameters_class();
+         $tasktemplate_content = UserTemplates::render(
+            Toolbox::addslashes_deep($tasktemplate->fields["content"]),
+            [
+               'ticket' => $parameters->getValues($this)
```

Comment:
Could be a change or problem as well. The parameter key should change to `change` or `problem` in those cases. If template can be shared between types, `itilobject` may be better so variables used in templates are valid across all types.

Reply:
Fixed.